### PR TITLE
Service class for the remaining 5 tables

### PR DESCRIPTION
**Pull request by Midhun-h4r**
#6 
Service class for the tables:
1.PickList
2.Login
3.Exam Answers
4.Exam Registration
5.Exam Schedule

### CODE REVIEW

**Review by rahulreghunath — CHANGES_REQUESTED** [review]
Change all variable name to camel case (eg $examSchedule)

File: app/Service/ExamAnswersService.php [hunk 1]
```diff
@@ -0,0 +1,98 @@
...
+    }
+
+    /**
+     * Delete ExamAnswers
+     * @param $id
+     * @return bool
+     */
+    public function delete($id)
+    {
+        $examAnswers = ExamAnswers::findOrFail($id);
```
Comment (rahulreghunath) [inline app/Service/ExamAnswersService.php:70]: if the data in the variable is single just use singular variable names as a good practice. Here change $examAnswers to $examAnswer

File: app/Service/ExamRegistrationService.php [hunk 2]
```diff
@@ -0,0 +1,96 @@
...
+class ExamRegistrationService extends Service
+{
+    /**
+     * Create ExamRegistration
+     * @param FormRequest $request
+     * @return int return inserted id if created else 0
+     */
+    public function create(FormRequest $request)
+    {
+        $ExamRegistration = new ExamRegistration();
```
Comment (rahulreghunath) [inline app/Service/ExamRegistrationService.php:27]: Naming changes again :sweat_smile:

**Review by rahulreghunath — CHANGES_REQUESTED** [review]

**Review by Midhun-h4r — COMMENTED** [review]
variable name corrected.

**Review by rahulreghunath — COMMENTED** [review]
Some more small changes

File: app/Service/LoginService.php [hunk 3]
```diff
@@ -0,0 +1,98 @@
...
+
+    /**
+     * @param int $id
+     * @param FormRequest|null $request
+     * @param null $options
+     * @return Model
+     */
+    public function update(int $id, FormRequest $request = null, $options = null)
+    {
+        $Login = $this->find($id);
```
Comment (rahulreghunath) [inline app/Service/LoginService.php:88]: :smile:

**Review by rahulreghunath — APPROVED** [review]

**Pull request merged**